.github/workflows/dotnet-selenium.yml

```diff
@@ -0,0 +1,39 @@
+name: Selenium tests
+
+on:
+  push:
+    branches: [ "CiCd" ]
```

Comment:
set up workflow dispatch to run tests when you want without commit

Reply:
Updated